docs/README.md

```diff
@@ -1,66 +0,0 @@
-# Conjur Website
-
-The web development environment uses Docker and Docker Compose to
-provide an expertly configured local Ruby environment. Instructions on
-how to install those are linked in the [top level
-README][dependencies].
-
-To build and serve the site on your computer, first run:
-
-```sh-session
-$ cd $(git rev-parse --show-cdup) # start in the project root
-$ docker-compose up -d docs
-```
-
-Then `open localhost:4000` to view it in your browser.
-
-The site is rebuilt automatically every time you save a file. Just
-refresh your browser tab (<kbd>&#8984;-R</kbd> on Mac, <kbd>Ctrl-R</kbd>
-elsewhere) to see updates.
-
-To stop the container, `docker-compose kill docs`.
-
-### Check for broken links
-
-```sh-session
-$ cd $(git rev-parse --show-cdup) # start in the project root
-$ ./checklinks.sh
-```
-
-This script uses Docker to run [html-proofer][proofer] on the rendered
-site and will report broken links as errors. It's automatically run as
-part of our continuous integration pipeline, but it's a good idea to
-run it yourself whenever you add or remove links.
-
-[dependencies]: ../README.md#Development_Dependencies "Development Dependencies"
-[proofer]: https://github.com/gjtorikian/html-proofer "HTML Proofer"
-
----
-
-### API Documentation
```

Comment:
Don't we want to keep the API docs in this project?

Reply:
This file is inside /docs/ so I am moving that part of the README into a new one inside /apidocs/ instead.